### ISSUE TITLE
Testing process stops

### ISSUE THREAD

**Issue by IagoAP** [serenity-bdd/serenity-core#1720]
Hello,

I'm having a problem, when I execute my serenity BDD system in a terminal I use the following command:

`mvn clean verify -Dmetafilter="groovy: (context == 'CONTEXT')"`

So, the test starts running and when the first one fails the process just stops, it doesn't close or start running the next test.

Here's the terminal state when it stop after the first failure:

           __  _____ _____ ____ _____   _____ _    ___ _     _____ ____  
  _       / / |_   _| ____/ ___|_   _| |  ___/ \  |_ _| |   | ____|  _ \ 
 (_)_____| |    | | |  _| \___ \ | |   | |_ / _ \  | || |   |  _| | | | |
  _|_____| |    | | | |___ ___) || |   |  _/ ___ \ | || |___| |___| |_| |
 (_)     | |    |_| |_____|____/ |_|   |_|/_/   \_\___|_____|_____|____/ 
          \_\                                                            

TEST FAILED WITH ERROR: Elaborar um pedido de KIT com promoção e empenho em DEPOSITO/NORMAL pagamento a vista
---------------------------------------------------------------------
[pool-112-thread-1] ERROR net.serenitybdd.core.Serenity - TEST FAILED AT STEP And Eu seleciono a promocao de valor ｟120,00｠
[pool-112-thread-1] ERROR net.serenitybdd.core.Serenity - Element <input class="ace radio-seleciona-promocao" name="form-field-radio-promocao" type="radio"> is not clickable at point (522,247) because another element <div class="blockUI blockOverlay"> obscures it

org.openqa.selenium.ElementClickInterceptedException: Element <input class="ace radio-seleciona-promocao" name="form-field-radio-promocao" type="radio"> is not clickable at point (522,247) because another element <div class="blockUI blockOverlay"> obscures it
Build info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T14:04:26.12Z'
System info: host: 'WORKSTATION', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.X', java.version: '1.8.0_211'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 67.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20190529130856, moz:geckodriverVersion: 0.24.0, moz:headless: true, moz:processID: 22456, moz:profile: /tmp/rust_mozprofile.s0ReSk..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, platformVersion: 4.15.X, rotatable: false, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 42612bda-29f1-4d94-afa4-6c0c4d053ad6
	(reflection-construct)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:276)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:83)
	at br.com.emp.pages.CPModal.selecionarPromocao(ConsultaPromocaoModal.java:62)
	at br.com.emp.steps.CPSteps.euSelecionoAPromocao(ConsultaProdutoSteps.java:97)
	(reflection-invoke)
	at org.jbehave.core.steps.StepCreator$ParametrisedStep.perform(StepCreator.java:733)
	at org.jbehave.core.embedder.PerformableTree$FineSoFar.run(PerformableTree.java:346)
...

(AfterStories)
[pool-113-thread-1] INFO net.thucydides.core.reports.json.JSONTestOutcomeReporter - Generating JSON report for Elaborar um pedido de KIT com promoção e empenho em DEPOSITO/NORMAL pagamento a vista to file /home/user/workspace_BDD/target/site/serenity/e79df21396dd4ee8e183dbf3db45fee4884eb7d5adb2266a65c625d1fdfcc321.json (using temp file /home/user/workspace_BDD/target/site/serenity/e79df21396dd4ee8e183dbf3db45fee4884eb7d5adb2266a65c625d1fdfcc321.json45c9b9cc-1160-4fb4-a496-a2f227b5bda5)

Generating reports view to '/home/user/workspace_BDD/target/jbehave' using formats '[stats, console, html, xml, serenityreporter]' and view properties '{decorateNonHtml=true}'
Reports view generated with 10 stories (of which 0 pending) containing 16 scenarios (of which 0 pending)
Failures in reports view: 1 scenarios failed

**Comment by wakaleo** [comment]
Can you provide a project that reproduces the issue?

**Comment by IagoAP** [comment]
Actually no, because when i run my project with less scenarios it runs smoothly, it'll be needed a project with around 73 stories and 132 scenarios.
I can't provide my company project witch is the one I'm having trouble with.
I searched a lot for problems running a lot of scenarios but could't find them, do u know of any related cases?

**Comment by wakaleo** [comment]
Does it stop suddenly after 300 seconds?

**Comment by IagoAP** [comment]
No

**Comment by IagoAP** [comment]
I let the process running all night and it's stops there like it's waiting for some I/O operation or a signal or something.

**Comment by wakaleo** [comment]
Could be a memory or resource issue, or some environment-specific issue. Without a reproducible sample, I could only look at it in the context of a support ticket. Does your company have a Serenity support contract?

**Comment by IagoAP** [comment]
I don't think so, I'll need to check it out, could take some time, I'll look into it.

Ty for the fast reply.

**Comment by IagoAP** [comment]
Hey @wakaleo, my company don't have a contract.
I solved it, when you sad it could be a memory or resource issue I did the following:
I made a shell script that erases my test files and copy there just the one i want to test and use it.
Then i made a shell who reads all the *.story files and call the other shell and execute the tests one by one and saves the results in a separate folder.
In the end i use one command I learned from one of your replies: mvn serenity:aggregate
Then i get all my tests done with one command.
I recompile the project every test, but i just compile it with one each time, i guess there is a loss of time there but i manage to execute all the tests.
